spec/chewy/config_spec.rb

```diff
@@ -55,32 +55,19 @@
     context 'nothing is defined' do
```

Comment:
Let's rewrite the whole `describe '#search_class'` block so it makes more sense in current situation with only one available option.